csi-driver-nfs/overlays/nerc-ocp-test/kustomization.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+commonLabels:
+  nerc.mghpcc.org/kustomized: "true"
+
+resources:
+- ../../base
+
+patches:
+- target:
+    kind: StorageClass
+    name: nfs-csi
+  patch: |
+    - op: replace
+      path: /parameters/server
+      value: nfs-server.nfs.svc.cluster.local
```

Comment:
Since we're always going to be using the in-cluster address -- which will be the same on very cluster -- should we just move this into `../../base`?

Reply:
I had it that way originally and changed it given that it's not necessarily a given that a cluster is going to use the local NFS deployment. For example, I'm looking into configuring NFS on the weka storage and using that with the csi-driver-nfs deployment on the prod cluster. That said, and as you mentioned, since a bunch of test clusters are going to use the locally-deployed NFS setup we can set the default to the locally deployed endpoint and patch as needed.

Reply:
I've updated the PR as requested.